flow-server/src/main/resources/webpack.generated.js

```diff
@@ -231,7 +231,7 @@ module.exports = {
     !devMode && new CompressionPlugin(),
 
     new ApplicationThemePlugin({
-      themeResourceFolder: path.resolve(flowFrontendFolder, 'theme'),
+      themeResourceFolder: path.resolve(flowFrontendFolder, 'themes'),
```

Comment:
```suggestion
      themeResourceFolder: path.resolve(flowFrontendFolder, 'theme'),
```
This matches `target/flow-frontend/theme/theme-generated.js` not `frontend/themes`